src/main/java/de/felixroske/jfxsupport/AbstractFxmlView.java

```diff
@@ -378,17 +385,17 @@ final String getFxmlPath() {
 	}
 
 	/**
-	 * Gets the resource bundle.
+	 * Gets the resource bundle or returns null.
 	 *
 	 * @param name
-	 *            the name
+	 *            the name of the resource bundle.
 	 * @return the resource bundle
 	 */
 	private ResourceBundle getResourceBundle(final String name) {
 		try {
 			return getBundle(name);
 		} catch (final MissingResourceException ex) {
-			LOGGER.debug("No resource bundle could be determined: " + ex.getMessage());
+			LOGGER.error("No resource bundle could be determined: " + ex.getMessage());
```

Comment:
Be aware that this is not an error:
There is simply no resource bundle specified.